behavior_metrics/configs/DL-torch.yml

```diff
@@ -21,6 +21,7 @@ Behaviors:
         Parameters:
             Model: 'model_pilotnet_torch_extreme.ckpt'
```

Comment:
The model listed here in the official repo is from last year and the predictions are not normalized. Are you using a new version that is different from that one? 
You should upload that version here two if so, please.